### ISSUE TITLE
Unable to create new entries on multi-site

### ISSUE THREAD

**Issue by tao** [statamic/cms#1152]
On a multi-site, if you create a new collection you are unable to see the new entry form until you update the collection.yaml file manually.

<img width="790" alt="Screenshot 2019-12-21 at 21 21 11" src="https://user-images.githubusercontent.com/1446331/71313903-d430f180-2437-11ea-8ec6-b854e44aa7fd.png">
 
The form is just blank as shown below:

<img width="792" alt="Screenshot 2019-12-21 at 21 22 29" src="https://user-images.githubusercontent.com/1446331/71313911-017d9f80-2438-11ea-9c13-02cdcb0300b8.png">

What you need to do is manually add the sites:
```
title: Books
route: 'books/{slug}'
blueprints:
  - book
revisions: false
sort_dir: asc
sites:
  - en
```

After that the form appears:

<img width="794" alt="Screenshot 2019-12-21 at 21 23 55" src="https://user-images.githubusercontent.com/1446331/71313928-34279800-2438-11ea-9305-bcadb6acce7a.png">

So some kind of warning might be necessary for multisite to update the config, or the default (or first?) site in `sites.php` could be added to new collections automatically so the form appears?

**Comment by peterkelm** [comment]
Same issue exists also for taxonomies,... on multi-site installs.

**Comment by damcclean** [comment]
Don't suppose you get any errors in the browser console?

**Comment by peterkelm** [comment]
@damcclean: see below error message:

```
[Error] TypeError: undefined is not an object (evaluating 't.findWhere(this.localizations,{active:!0}).livePreviewUrl')
livePreviewUrl — PublishForm.vue:350
(anonyme Funktion) — vue.common.prod.js:6:26601
(anonyme Funktion) — vue.common.prod.js:6:27746
(anonyme Funktion) — vue.common.prod.js:6:29650
fn — PublishForm.vue:7
r — vue.common.prod.js:6:15526
(anonyme Funktion) — vue.common.prod.js:6:35314
r — vue.common.prod.js:6:68006
(anonyme Funktion) — vue.common.prod.js:6:26601
pn — vue.common.prod.js:6:26514
(anonyme Funktion) — vue.common.prod.js:6:68027
init — vue.common.prod.js:6:19343
(anonyme Funktion) — vue.common.prod.js:6:56349
d — vue.common.prod.js:6:56624
p — vue.common.prod.js:6:57213
d — vue.common.prod.js:6:56739
(anonyme Funktion) — vue.common.prod.js:6:60762
(anonyme Funktion) — vue.common.prod.js:6:34374
r — vue.common.prod.js:6:68004
(anonyme Funktion) — vue.common.prod.js:6:26601
pn — vue.common.prod.js:6:26514
(anonyme Funktion) — vue.common.prod.js:6:68027
init — vue.common.prod.js:6:19343
(anonyme Funktion) — vue.common.prod.js:6:56349
d — vue.common.prod.js:6:56624
(anonyme Funktion) — vue.common.prod.js:6:60762
(anonyme Funktion) — vue.common.prod.js:6:34374
r — vue.common.prod.js:6:68004
(anonyme Funktion) — vue.common.prod.js:6:26601
pn — vue.common.prod.js:6:26514
(anonyme Funktion) — vue.common.prod.js:6:68027
init — vue.common.prod.js:6:19343
(anonyme Funktion) — vue.common.prod.js:6:56349
d — vue.common.prod.js:6:56624
p — vue.common.prod.js:6:57213
d — vue.common.prod.js:6:56739
p — vue.common.prod.js:6:57213
d — vue.common.prod.js:6:56739
p — vue.common.prod.js:6:57213
d — vue.common.prod.js:6:56739
p — vue.common.prod.js:6:57213
d — vue.common.prod.js:6:56739
(anonyme Funktion) — vue.common.prod.js:6:61030
(anonyme Funktion) — vue.common.prod.js:6:34374
r — vue.common.prod.js:6:68004
(anonyme Funktion) — vue.common.prod.js:6:26601
pn — vue.common.prod.js:6:26514
(anonyme Funktion) — vue.common.prod.js:6:68027
(anonyme Funktion) — vue.common.prod.js:6:32873
Mn — vue.common.prod.js:6:30202
start — Statamic.js:81
start
Globaler Code — en:241

	Be (vendor.js:1:1080300)
	qe (vendor.js:1:1080213)
	$e (vendor.js:1:1079863)
	(anonyme Funktion) (vendor.js:1:1103297)
	r (vendor.js:1:1136651)
	(anonyme Funktion) (vendor.js:1:1095240)
	pn (vendor.js:1:1095157)
	(anonyme Funktion) (vendor.js:1:1136664)
	init (vendor.js:1:1087992)
	(anonyme Funktion) (vendor.js:1:1124962)
	d (vendor.js:1:1125237)
	p (vendor.js:1:1125826)
	d (vendor.js:1:1125352)
	(anonyme Funktion) (vendor.js:1:1129375)
	(anonyme Funktion) (vendor.js:1:1102312)
	r (vendor.js:1:1136641)
	(anonyme Funktion) (vendor.js:1:1095240)
	pn (vendor.js:1:1095157)
	(anonyme Funktion) (vendor.js:1:1136664)
	init (vendor.js:1:1087992)
	(anonyme Funktion) (vendor.js:1:1124962)
	d (vendor.js:1:1125237)
	(anonyme Funktion) (vendor.js:1:1129375)
	(anonyme Funktion) (vendor.js:1:1102312)
	r (vendor.js:1:1136641)
	(anonyme Funktion) (vendor.js:1:1095240)
	pn (vendor.js:1:1095157)
	(anonyme Funktion) (vendor.js:1:1136664)
	init (vendor.js:1:1087992)
	(anonyme Funktion) (vendor.js:1:1124962)
	d (vendor.js:1:1125237)
	p (vendor.js:1:1125826)
	d (vendor.js:1:1125352)
	p (vendor.js:1:1125826)
	d (vendor.js:1:1125352)
	p (vendor.js:1:1125826)
	d (vendor.js:1:1125352)
	p (vendor.js:1:1125826)
	d (vendor.js:1:1125352)
	(anonyme Funktion) (vendor.js:1:1129643)
	(anonyme Funktion) (vendor.js:1:1102312)
	r (vendor.js:1:1136641)
	(anonyme Funktion) (vendor.js:1:1095240)
	pn (vendor.js:1:1095157)
	(anonyme Funktion) (vendor.js:1:1136664)
	(anonyme Funktion) (vendor.js:1:1100823)
	Mn (vendor.js:1:1097702)
	start (app.js:1:1535424)
	start
	Globaler Code (en:241)
```

**Comment by jackmcdade** [comment]
Yeah clearly we need to loop back around to multi-site :)

**Comment by jasonvarga** [comment]
Fixed in 33b398e2
